### PR TITLE
Feature rf gu 1

### PR DESCRIPTION
**Pull request by IreneGaita**

### CODE REVIEW

File: Website/flaskr/gestione_adozioni/GestioneAdozioniService.py [hunk 1]
```diff
@@ -31,6 +31,11 @@ def get_alveari():
     return Alveare.query.all()
 
 
+def get_AlveariByApicoltore(id_apicoltore):
```
Comment (Luigi11162) [inline Website/flaskr/gestione_adozioni/GestioneAdozioniService.py]: Rimuovere metodo ridondante

File: Website/flaskr/gestione_adozioni/GestioneAdozioniService.py [hunk 2]
```diff
@@ -42,6 +47,15 @@ def affitto_alveare(ticket, percentuale):
     db.session.add(ticket)
     db.session.commit()
     decrementa_percentuale(ticket.id_alveare, percentuale)
+
+def update_stato(id, covata_compatta, popolazione, polline, stato_cellette):
```
Comment (Luigi11162) [inline Website/flaskr/gestione_adozioni/GestioneAdozioniService.py]: cambiare in aggiorna_stato

**Review by Luigi11162 — CHANGES_REQUESTED** [review]

**Review by Luigi11162 — APPROVED** [review]

File: Website/flaskr/templates/inserimento_alveare.html [hunk 3]
```diff
@@ -1,109 +1,211 @@
...
+                            name="prezzo"
+                            type="number"
+                            required=true
+                            placeholder="Inserisci prezzo"
+                    />
+                  </div>
+                </div>
+                     <div class="col-md-6 mb-4">
+                  <div class="form-outline">
+                      <label for="covata_compatta">La covata è compatta?</label><br>
```
Comment (andesrule) [inline Website/flaskr/templates/inserimento_alveare.html:130]: Press ctrl+alt+L to indent files

File: Website/flaskr/templates/inserimento_alveare.html [hunk 4]
```diff
@@ -1,109 +1,211 @@
...
+                    <label for="covata_compatta">si</label>
+                    <input type="radio" id="covata_compatta2" name="covata_compatta" value="0">
+                    <label for="covata_compatta">no</label>
+                  </div>
+                     </div>
+                  </div>
+                  <div class="row">
+                <div class="col-md-6 mb-4">
+                  <div class="form-outline">
+                      <label for="tipo_fiore">Inserire il nome del fiore impollinato: </label>
```
Comment (andesrule) [inline Website/flaskr/templates/inserimento_alveare.html:141]: Press ctrl+alt+L to indent files

**Review by andesrule — COMMENTED** [review]
Press ctrl+alt+L to indent files

**Pull request merged**